TACC/tup-ui
# Task/TUP-365 create ticket form modal

## Overview:
Replicated the create ticket modal/form from Core-Portal

## Related:

- [TUP-365](https://jira.tacc.utexas.edu/browse/TUP-365)

## Changes:

## Testing:

1. Go to http://localhost:8000/dashboard/
2. Click on the + New Ticket button above the the tickets table
3. Test out the form fields
4. Click on the Add Ticket button to submit the form
5. Verify that a success message appears in the modal footer with the newly created ticket ID
6. Verify that the form data you've entered is correctly displayed back by viewing your ticket in Core-Portal (e.g. https://frontera-portal.tacc.utexas.edu/workbench/dashboard)

## UI:

## Notes:
You'll noticed that the last character of description text you've entered in the form gets trimmed when you view it in your ticket in Core-Portal.
This is caused by a difference in how our tup-services backend formats the description text prior to sending it to RT and after retrieving it from RT vs. how Core-Portal does the same.
